packages/models/files.js

```diff
@@ -7,6 +7,12 @@ const Files = new FilesCollection({
     debug: true,
     collectionName: 'files',
     protected(fileObj) {
+        var session = this.params.query.xmtok;
+        var userId  = (Meteor.server.sessions[session] && Meteor.server.sessions[session].userId) ? Meteor.server.sessions[session].userId : null;
```

Comment:
@menelike 100% Agree, tried to found how to use it to get and login token from it to find a user who uses it in Mongo. Can't find that piece of code, lost in my archives. There is some undocumented method on Accounts package....